storage/container_test.go

```diff
@@ -0,0 +1,127 @@
+package storage
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/go-oracle-terraform/helper"
+	"github.com/hashicorp/go-oracle-terraform/opc"
+)
+
+const _ContainerName = "test-str-container"
+const _ContainerURLKey = "test-url-key"
+const _ContainerURLKey2 = "test-url-key2"
+const _ContainerMaxAge = 50
+
+func TestAccContainerLifeCycle(t *testing.T) {
+	helper.Test(t, helper.TestCase{})
+
+	client, err := getStorageTestClient(&opc.Config{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	readACLs := []string{"test-read-acl1", "test-read-acl2"}
+	writeACLs := []string{"test-write-acl1", "test-write-acl2"}
+	allowedOrigins := []string{"allowed-origin-1", "allowed-origin-2"}
+
+	createContainerInput := CreateContainerInput{
+		Name:           _ContainerName,
+		ReadACLs:       readACLs,
+		WriteACLs:      writeACLs,
+		URLKey:         _ContainerURLKey,
+		URLKey2:        _ContainerURLKey2,
+		AllowedOrigins: allowedOrigins,
+	}
+
+	createdContainer, err := client.CreateContainer(&createContainerInput)
+	if err != nil {
+		t.Fatal(err)
+	}
+	log.Printf("Successfully created Container: %+v", createdContainer)
+	defer deleteContainer(t, client, _ContainerName)
+
+	getContainerInput := GetContainerInput{
+		Name: _ContainerName,
+	}
+	container, err := client.GetContainer(&getContainerInput)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !reflect.DeepEqual(container.ReadACLs, readACLs) {
```

Comment:
instead of `reflect.DeepEqual`, possible to use `pretty.Compare`? The print output on a failed `reflect.DeepEqual` is fairly hard to read compared to the diff from `pretty.Compare`